nhrpd/nhrp_peer.c

```diff
@@ -1063,7 +1064,8 @@ static void nhrp_peer_forward(struct nhrp_peer *p,
 		nhrp_ext_complete(zb, dst);
 	}
 
-	nhrp_packet_complete(zb, hdr);
+	// XXX: auth already handled ???
```

Comment:
What is the answer to this question?

Reply:
Packet is already authenticated in nhrp_peer_recv() before forwarding. Removed comment